### ISSUE TITLE
Switching app.component.css to scss file breaks webpack build

### ISSUE THREAD

**Issue by JeromeJoestar** [DevWurm/angular-2-electron-seed#13]
When changing the app.component.css file to a sass file I run into the following error:

```
ERROR in ./src/app/app.component.scss
Module parse failed: /Users/mparker/Projects/Client Projects/wanu-desktop/src/app/app.component.scss Unexpected token (1:0)
You may need an appropriate loader to handle this file type.
.test{
     color: #000;
 }
```

I have the original config structure for webpack, I am confused on what change I need to make.
 :/

**Comment by nattu1989** [comment]
Have the same problem. Would really appreciate any help

**Comment by DevWurm** [comment]
We already had an issue regarding this, please have a look at #8.

**Comment by JeromeJoestar** [comment]
Hmm I changed all the styles.css references to styles.scss and am still having this issue.

**Comment by JeromeJoestar** [comment]
Actually I just went through the config file and noticed that there was a missing input/export object for the css loader after I cloned the project. Adding that in from the commit at the issue you referenced seems to have done the trick. @nattu1989 go to the issue mentioned above and look for the commit change css references to scss. Take the webpack config code and copy/paste that.
Everything is working for me now.
